test/integration/targets/connection_aws_ssm/runme.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+# Setup Environment
+ansible-playbook -c local aws_ssm_integration_test_setup.yml "$@"
+
+# Execute test cases only on Python 2.7
+if [ -e aws-env-vars.sh ]; then
```

Comment:
This comment and conditional are out-of-date now that the tests are supporting Python 3 also.

Reply:
Fixed in https://github.com/ansible/ansible/pull/49652/commits/d69abb2abcc0c00912ae786c9f2a81bf489073a7